client/core/core.go

```diff
@@ -3084,6 +3082,13 @@ func (c *Core) checkUnpaidFees(dcrWallet *xcWallet) {
 // called if the client was shutdown after a fee was paid, but before it had the
 // requisite confirmations for the 'notifyfee' message to be sent to the server.
 func (c *Core) reFee(dcrWallet *xcWallet, dc *dexConnection) {
+
+	// Return if the coin is already in blockWaiters.
+	regFeeAssetID, _ := dex.BipSymbolID(regFeeAssetSymbol)
+	if c.existsWaiter(coinIDString(regFeeAssetID, dc.acct.feeCoin)) {
```

Comment:
Looking back I realized this isn't perfect since different assets can have the same coinid but that's super unlikely to happen.  Also, it really doesn't need to be converted with `coinIDString` to have a map key as we could have just done hex.EncodeToString on the feeCoin.
Regardless, I'm flagging this PR as part of the [0.1.4 milestone](https://github.com/decred/dcrdex/milestone/10).